anaconda_anon_usage/tokens.py

```diff
@@ -76,16 +118,24 @@ def token_string(prefix=None, enabled=True):
     appended to the conda user agent.
     """
     parts = ["aau/" + __version__]
-    if enabled:
+    if enabled or system_token():
```

Comment:
Let's add a comment here that this is a departure from the current way activation of aau works, given that it will have a big exception to the previously agreed to consent. I'm thinking about those that have actively disabled it in the past year basically.

Maybe for clarity this condition could be moved into a new `is_active` function that would take user intent and existence of an org token into account?

Reply:
Good call

Reply:
I have moved this logic into `patch.py` and added a comment to reduce surprise. Note that the use case here is for a system administrator to effectively "force" telemetry with the appending of an organization token. If a user does not have an organization token installed, their `anaconda_anon_usage` flag behavior is fully respected.